src/r2mm/downloading/BetterThunderstoreDownloader.ts

```
@@ -86,35 +87,23 @@ export default class BetterThunderstoreDownloader extends ThunderstoreDownloader
                            ignoreCache: boolean,
                            callback: (progress: number, modName: string, status: number, err: R2Error | null) => void,
                            completedCallback: (modList: ThunderstoreCombo[]) => void) {
+
+        const modList = await ProfileModList.getModList(profile);
```

Good idea to fail early :+1: